plaso/engine/artifacts_filter_file.py

```diff
@@ -0,0 +1,272 @@
+# -*- coding: utf-8 -*-
+"""Filter file."""
+
+from __future__ import unicode_literals
+
+import logging
+import re
+
+from artifacts import definitions as artifact_types
+from artifacts import errors as artifacts_errors
+from artifacts import reader as artifacts_reader
+from artifacts import registry as artifacts_registry
+
+from dfvfs.helpers import file_system_searcher
+from dfwinreg import registry_searcher
+
+
+from plaso.lib import py2to3
+from plaso.lib import errors
+
+
+ARTIFACTS_FILTER_FILE = 'ARTIFACTS_FILTER_FILE'
+COMPATIBLE_DFWINREG_KEYS = [r'HKEY_LOCAL_MACHINE',
```

Comment:
for code consistency please line break after [, replace r' by regular ' (escape \)

Reply:
Done